### DISCUSSION TITLE
Can we all have Yarn Add command side-by-side to Npm Install?

### DISCUSSION THREAD

**Discussion by nbaua** [npm/feedback#262] [Website] [answered]
Hi,

I know this is something not easy to ask?
Still seeing that many users like me prefer Yarn Add over the NPM install, so can we all have Yarn Add command side-by-side to Npm Install? We all know, you'd be the BIG brother always, however it would be good to have the Yarn Add command side by side.

![image](https://user-images.githubusercontent.com/17998763/110196276-7e444000-7e69-11eb-9df8-059e84ec6545.png)

You know, Adding this will still not harm anyone, in fact people will respect NPM more. Thanks.

Appreciate for making NPM, would appreciate more, if this is taken care of.

**Comment by MylesBorins** [comment, marked as answer]
@nbaua I don't think this is a request that we will be able to accommodate. There are a number of package managers including `npm`, `yarn`, `pnpm`... and we can't include every single package manager install instructions on the npm page.

If you take a look at [yarnpkg.com](https://yarnpkg.com/package/react-native-video) you'll notice that they have the `yarn add` instruction but not `npm install`, which I think is the right decision. Making an uncrowded design is hard and if they tried to include all the ways to install a package it would definitely start to look way too busy.

Thanks for the suggestion!
